.github/workflows/create-daily-docs-sync-pr.yaml

```diff
@@ -12,8 +12,30 @@ env:
   GITHUB_REPOSITORY_OWNER: solidity-docs
 
 jobs:
+  checkSolidityRelease:
+    runs-on: ubuntu-latest
+    outputs:
+      current_release: ${{ steps.current-release-version.outputs.current_release }}
+      latest_release: ${{ steps.latest-release-version.outputs.latest_release }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get current Solidity docs version
+        id: current-release-version
+        run: |
+          current_release=$(<latest_solidity.release)
+          echo "current_release=$current_release" >> $GITHUB_OUTPUT
+      - name: Get latest Solidity release version
+        id: latest-release-version
+        run: |
+          latest_release=$(
+            curl -sL https://api.github.com/repos/ethereum/solidity/releases/latest | jq --raw-output ".tag_name" | cut -d"v" -f2
+          )
+          echo "latest_release=$latest_release" >> $GITHUB_OUTPUT
+
   createPullRequest:
     runs-on: ubuntu-latest
+    needs: checkSolidityRelease
+    if: needs.checkSolidityRelease.outputs.current_release != needs.checkSolidityRelease.outputs.latest_release
```

Comment:
The sync PR description also says that it's perfectly safe to close a sync PR and wait for the next one and with this change it's no longer true, which will create problems for translators.

The description of the sync PR should also be updated to say that sync PRs are generated when there's an untranslated release and no pending sync PR (rather than in regular intervals as until now) so that they know when to expect one.